vgrep.go

```diff
@@ -465,7 +465,7 @@ func dispatchCommand(input string) bool {
 			return false
 		}
 		for _, idx := range indices {
-			commandShow(idx)
+			return commandShow(idx)
```

Comment:
I think we should not change the error case to return true as all other commands don't do that either... This would lead to quitting from the interactive mode if a wrong index was specified.

How about adding a `return false` just after the loop? We could also factor that return out of the error case just above the loop and putting the iteration over the indices in an `else` branch.
```go
for _, idx := range indices {
    commandShow(idx)
}
return false
```

Reply:
> I think we should not change the error case to return true as all other commands don't do that either... This would lead to quitting from the interactive mode if a wrong index was specified.

That's a fair point.

> How about adding a `return false` just after the loop? We could also factor that return out of the error case just above the loop and putting the iteration over the indices in an `else` branch.

Sounds good, thanks!